### ISSUE TITLE
Listening to db.changes since:'now' returns all changes

### ISSUE THREAD

**Issue by tudordumitriu** [pouchdb/pouchdb#7164]
### Issue
Hi all
We have the following setup: a remote CouchDB synced (filtered) to a local PouchDB and we need to listen to latest changes, which "since:'now'" param should handle - pretty common scenario.
Unfortunately, it seems to get ALL changes that the user did from the very beginning and I would expect to only trigger when new changes happen (I am wrong?).

### Info
- Environment: Browser ( "pouchdb": "^6.4.3",    "pouchdb-authentication": "^1.1.2")
- Platform: Chrome
- Adapter: IndexedDB
- Server: CouchDB

### Code
```
import PouchDB from "pouchdb";
import PouchAuthentication from "pouchdb-authentication";

PouchDB.plugin(PouchAuthentication);

var remoteDB = new PouchDB("http://192.168.0.127:5984/test", {
    skip_setup: true
  }),
  localDB = null,
  userId = null;

export const initSync = id => {
  userId = id;
};

export const listenToChanges = async callback => {
    //this is needed because after login we need a clean db for the new user
    localDB = new PouchDB("test");
    return localDB.destroy().then(() => {
      localDB = new PouchDB("test");
      localDB
        .sync(remoteDB, {
          live: true,
          retry: true,
          filter: "app/userFilter",
          query_params: { userID: userId }
        })
        .on("error", console.log.bind(console));

        localDB
            .changes({
                live: true,
                since: 'now',           
            })
            .on('change', function (change) {
                callback && callback();
                console.log('change', change);
            }).on('complete', function (info) {
                console.log('complete', info);
            }).on('error', function (err) {
                console.log('err', err);
            });
    });
};
```

**Comment by tudordumitriu** [comment]
I think I found something helpful, before starting the longpoll changes call I see 2 other changes calls on the network and the first one has **since: 0**
http://192.168.0.127:5984/test/_changes?style=all_docs&seq_interval=100&filter=app%2FuserFilter&userID=org.couchdb.user%3Az&since=0&limit=100

**Comment by tudordumitriu** [comment]
I think I might have found a solution for my problem but not sure if this isn't still a bug.

I also understand that this "might" be correct behavior considering the PouchDb gets destroyed and recreated, and we are listening to changes on the localDB (NOT REMOTE ONE), so maybe I am using / understanding this the wrong way.
But the question remains, how could we listen ONLY to NEW changes on the locally replicated DB? 

So what I managed to to is to trigger listening to changes only once the Sync is complete.
```
localDB
            .sync(remoteDB, {
                live: true,
                retry: true,
                filter: "app/userFilter",
                query_params: { userID: userId }
            })
            .on('change', function (change) {
                console.log('replication changes', change);
            }).on('paused', function (info) {
                console.log('pause', syncFinished);
                if (syncFinished) {
                    localDB
                        .changes({
                            live: true,
                            since: 'now',
                            //include_docs: true
                        })
                        .on("change", change => {
                            console.log('change', change);
                            //callback && callback();
                        });
                }
            }).on('active', function (info) {
                console.log('active', info);
                syncFinished = true;
            });
```

**Comment by daleharvey** [comment]
Thats exactly by design, since: now does a db.info() and uses the current update_seq of the database, the changes call on the local database has no idea what the state of your remote database is. Your fix looks like the correct one

**Comment by elilambnz** [comment]
After experiencing some unexpected behaviour with the above example, I've come up with an alternative solution for anyone else who may also be running into the same issues.

1. The first database change is always missed by the `.changes()` event emitter
2. The `change` event is called more than once for a changed doc

In my case, the first issue was due to the `syncFinished` flag not being set until after the first change was made, so the event emitter is not yet initialized. In my solution I have removed this condition. The second issue was due to the `paused` event being called often, which caused an abundance of `change` events. In my solution, I initialise the `.changes()` event emitter outside of the `.sync`.

```js
localDB
  .changes({
    live: true,
    since: 'now',
    include_docs: true,
  })
  .on('change', info => handleChange(info.doc))

localDB.sync(remoteDB, {
  live: true,
  retry: true,
  ...
})
```
